internal/dbusservice/methods.go

```diff
@@ -68,3 +72,8 @@ func (s *Service) UserPreCheck(username string) (userinfo string, dbusErr *dbus.
 	}
 	return userinfo, nil
 }
+
+// makeCanceledError creates a dbus.Error for a canceled operation.
+func makeCanceledError() *dbus.Error {
+	return &dbus.Error{Name: "com.ubuntu.authd.Canceled"}
```

Comment:
Nice, maybe we want to be even more specific?


```suggestion
	return &dbus.Error{Name: "com.ubuntu.authd.IsAuthenticated.Canceled"}
```

Reply:
Yeah, I had similar thinking, just mentioned in case... But you can go with this one